tests/test_tools_manually.yml

```diff
@@ -7,19 +7,33 @@
     - variant_calling
   files:
     - path: results/variant_calling/ascat/sample4_vs_sample3/sample4_vs_sample3.after_correction_gc_rt.sample4_vs_sample3.tumour.germline.png
+      md5sum: 4ac774bf5f1157e77426fd82f5a03001
```

Comment:
Maybe a stupid question, but I find it very odd that the checksums for the ascat files are consecutive 🤔 
Could you double-check this please? Might also do that.

Reply:
Thanks for pointing out that mistake out 👍👏  Those "consecutive" md5sums are just placeholders which I used when trying to find the correct md5sums.  Since all the tests run byt GHA were passing, it must means that the CI-test `test_tools_manually.yml` (or at least the test with the tags `ascat_manual`, `manual`, `somatic`, and `variant_callingat part`) is not executed by GHA 🤔  I guess that is due to the following list not containing any of those aforementioned tags:

https://github.com/nf-core/sarek/blob/89d15744e2410e03c86be664372b6cc06a8ac97f/.github/workflows/ci.yml#L33-L63

I don't know if some of those four tags should be included in the above list.

I'll see if I can run the test locally, and try to get some valid md5sums for the test.

Reply:
Okay great! Yes, all yml files that have a manual tag should not be run on GHA because they need either too much time, memory or space for the testdata. 
The md5sums I get for the ascat somatic run are unfortunately unstable due to timestamps. I can send you some `contains` strings that you could add.

Reply:
@SusiJo : Thanks, that would be great 🙏